flask/app/participants.py

```diff
@@ -278,28 +320,38 @@ def update_participant(id: int):
 @login_required
 @check_admin
 def create_participant():
+    app.logger.debug("Checking request body")
     if not request.json:
+        app.logger.error("Request body is not JSON")
         abort(415, description="Request body must be JSON")
-
+    app.logger.debug("Request body is JSON")
     try:
+        app.logger.debug(
+            "Assigning updated and created by IDs to user ID '%s'", current_user.user_id
+        )
         updated_by_id = current_user.user_id
         created_by_id = current_user.user_id
     except:  # LOGIN_DISABLED
+        app.logger.debug(
+            "LOGIN_DISABLED = True so updated and created by IDs are defaulting to 1"
+        )
         updated_by_id = 1
         created_by_id = 1
 
     # check if the participant exists under a given family
-
+    app.logger.debug("Checking if participant already exists under family")
     ptp_query = models.Participant.query.filter(
         models.Participant.family_id == request.json.get("family_id"),
         models.Participant.participant_codename
         == request.json.get("participant_codename"),
     )
 
     if ptp_query.first() is not None:
+        app.logger.error("Participant codename already exists under family")
```

Comment:
I'm thinking from here to 377 values being queried could be logged for more granularity. something like `app.logger.error("Participant codename '%s' already exists under family ID", request.json.get("participant_codename")`.